src/components/AttachmentViewer/components/InteractiveImage/InteractiveImage.tsx

```diff
@@ -55,15 +57,18 @@ const InteractiveImage = ({
       backgroundImage={src}
       data-testid="zoomed-interactive-image"
     >
-      <Image
-        className={className}
-        zoom={zoomFactor}
-        loading="lazy"
-        src={src}
-        alt={alt}
-        data-testid="interactive-image"
-        {...rest}
-      />
+      <Wrapper>
+        <Image
+          className={className}
+          zoom={zoomFactor}
+          loading="lazy"
+          src={src}
+          alt={alt}
+          data-testid="interactive-image"
+          {...rest}
+        />
+        {caption && <StyledFigCaption>{caption}</StyledFigCaption>}
```

Comment:
the caption needs to be under the image. Why not make Wrapper a figure element instead of div?currenly you always need to put interactiveImage within a figure element, they should be one thing instead

Reply:
Did it (changed Wrapper to a figure) but that doesn't solve anything.